redfish_utilities/inventory.py

```diff
@@ -207,7 +207,9 @@ def catalog_resource( context, resource, inventory, chassis_id ):
         "AssetTag": resource.get( "AssetTag", None ),
         "Label": resource.get( location_prop, {} ).get( "PartLocation", {} ).get( "ServiceLabel", None ),
         "State": resource.get( "Status", {} ).get( "State", None ),
-        "Description": None
+        "Description": None,
+        "ContainedByChassisUri": None,
```

Comment:
I think we can move these new items up to line 60, and simply look at Contains/ContainedBy when building the outer chassis object. A component within a chassis should be placed in the right chassis object, which means you shouldn't need to track the containment path at the component level.